pkg/search/postgres/common.go

```diff
@@ -753,6 +754,53 @@ func retryableRunGetManyQueryForSchema(ctx context.Context, query *query, db *pg
 	return results, nil
 }
 
+// RunGetManyQueryForSchemaType executes a request for just the search against the database and unmarshal it to given type.
```

Comment:
Actually, coming back here, there's also a bunch of code duplication here. Maybe rewrite `retryableRunGetManyQueryForSchema` in a callback fashion, taking a `callback func([]byte) error`? This should avoid the `[][]byte` allocation in the proto case. Note that you _need_ to shift it to the inner `retryable...` function, as callbacks and retries don't go too well together.

Reply:
I remove this functions here https://github.com/stackrox/stackrox/pull/3454
Let me merge this PRs so diff will be easier to read